docs/locators.md

```diff
@@ -0,0 +1,289 @@
+      key: "startPage.navButton.text"
+    },
+    controlType: "sap.m.Button",
+    viewName: "test.Sample.view.Main"
+  }
+}
+
+// get it
+const control = await browser.asControl(identifier)
+// use it
+await control.press()
+```
+
+Here are couple of samples to get you comfortable with `wdi5`'s/OPA5's locators/matchers.
+
+?> Note: all of them are essentially taken from the `/examples/ui5-js-app/webapp/test/e2e/**/*.test.js` files.
+
+## ancestor
+
+Get a "child" control from a "parent" (ancestor).
+
+```javascript
+it("get the child", async () => {
+  const ancestorSelector = {
+    selector: {
+      controlType: "sap.m.Title",
+      ancestor: {
+        controlType: "sap.m.Panel"
+      }
+    }
+  }
+
+  const text = await browser.asControl(ancestorSelector).getText()
+
+  expect(text).toEqual("Custom Toolbar with a header text")
+})
+```
+
+## bindingPath
+
+Retrieve a UI5 control that has a binding context with this exact binding path.  
+Typically not used standalone, but in combination with [the `controlType` locator](#controlType).
+
+```javascript
+it("select ui5 input control by binding and set new value (w/ custom wdio_ui5_key)", async () => {
+  const inputSelector = {
+    selector: {
+      interaction: "focus",
+      bindingPath: {
+        propertyPath: "/Customers('TRAIH')/ContactName"
+      },
+      viewName: "test.Sample.view.Main",
+      controlType: "sap.m.Input"
+    }
+  }
+
+  const value = "new value"
+  const input = await browser.asControl(inputSelector)
+  await input.enterText(value)
+
+  expect(await input.getValue()).toEqual(value)
+})
+```
+
+## controlType
+
+Specify a UI5 control by namespace for retrieval.  
+Typically not used standalone, but in combination with any other locator.
+
+```javascript
+it("get a specific list item", async () => {
+  const selectorWithTitleProperty = {
+    selector: {w
+      controlType: "sap.m.StandardListItem",
+      viewName: "test.Sample.view.Other",
+      properties: {
+        title: "Margaret Peacock"
+      }
+    }
+  }
+  const titleFromListItem = await browser.asControl(selectorWithTitleProperty).getTitle()
+  expect(titleFromListItem).toEqual("Margaret Peacock")
+})
+```
+
+## descendant
+
+Get a "parent" with the help of a "child" (aka the descendant) control.
+
+```javascript
+it("get the parent", async () => {
+  const descendantSelector = {
+    selector: {
+      controlType: "sap.m.Panel",
+      descendant: {
+        controlType: "sap.m.Title",
+        properties: {
+          text: "Custom Toolbar with a header text"
+        }
+      }
+    }
+  }
+  const sPanelText = await browser.asControl(descendantSelector).getHeaderText()
+  expect(sPanelText).toEqual("Header Text")
+})
+```
+
+## i18n
+
+Retrieve a control by it's internationalization key rather than by its' visible text.
+
+```javascript
+it("check i18nText matcher user button", async () => {
+  const i18nSelector = {
+    selector: {
+      i18NText: {
+        propertyName: "text",
```

Comment:
the locator `i18nText` actually as more than those two parameters but I think we currently only support `propertyName` and `key` yet

Reply:
right on, prepped a dedicated test file